packages/react-router-dom/__tests__/data-static-router-test.tsx

```diff
@@ -1,13 +1,9 @@
 import * as React from "react";
 import * as ReactDOMServer from "react-dom/server";
-import {
-  Route,
-  Outlet,
-  useLoaderData,
-  useLocation,
-  useMatches,
-} from "../index";
-import { DataStaticRouter } from "../server";
+import { Route, useLocation } from "react-router-dom";
+import { DataStaticRouter } from "react-router-dom/server";
+import { Outlet } from "react-router/lib/components";
+import { useLoaderData, useMatches } from "react-router/lib/hooks";
```

Comment:
The static routes aren't currently exported through the main `index.ts` (carried that behavior forward from 6.3.0).  I don't see an issue re-exporting through `index.ts` since it'll tree shake away but would probably want to clarify there's not another reason for that type of setup

Reply:
We're good here, I think this comment was on an outdated local copy, everything except `/server` is coming from the root package now 👍